app/views/npq_separation/admin/schools/index.html.erb

```diff
@@ -1,18 +1,49 @@
-<h1 class="govuk-heading-l">All schools</h1>
+<h1 class="govuk-heading-l">Workplaces</h1>
+<p class="govuk-body">Search for a workplace to find its:</p>
+<ul class="govuk-list govuk-list--bullet">
+  <li>ID</li>
+  <li>Unique Reference Number (URN)</li>
+  <li>UK Provider Reference Number (UKPRN)</li>
+  <li>local authority</li>
+  <li>address</li>
+</ul>
+
+<div style="background-color: #f3f2f1; padding: 20px 15px; margin-bottom: 30px;">
+  <%= form_with method: :get do |f| %>
+    <%= f.govuk_text_field(
+          :q,
+          value: params[:q],
+          hint: { text: "Search by entering the workplace name or its Unique Reference Number (URN)" },
+          label: { text: "Find a workplace", tag: "h1", size: "m" },
```

Comment:
Why H1, there's already an H1 on the page? I'd expect an `m` should be a h3 (or possible h2) ?